README.md

```diff
@@ -29,6 +29,21 @@ To start Jekyll server locally.
 5. Make some changes, refresh your browser to preview them.
 6. Submit a pull request.
 
+## New release
+
+For each new release we keep stable documentation tree for the record.
+
+### Foreman Core
+
+1. cp -r manuals/X.Y manuals/X.Z
+2. edit file _layouts/manual.html and modify "latestversion"
```

Comment:
> That's only changed after release else people get the 1.10 manual while 1.9's stable.  (http://projects.theforeman.org/projects/foreman/wiki/Release_Process#Completion-tasks)
> 
> When copying the manual, also:
> 
> 1. change version numbers in manuals/1.10/*.md
> 1. add version to documentation.md

Ok updated, if you have no more comments on the warning box scheme, we
are ready to merge this one. @stbenjam any other comments?

-- 
Later,
 Lukas #lzap Zapletal